recipes/glaes/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "glaes" %}
+{% set version = "1.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FZJ-IEK3-VSA/glaes/archive/v{{ version }}.tar.gz
+  sha256: dd2ad7520445d5a62e734a746003ff260aa19df7068f194b6112695c0ea3f845
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - geokit >=1.2.6
+    - gdal >2.0.0,<3.0.0
+    - numpy
+    - descartes
+    - pandas
+    - scipy
+    - matplotlib-base
+    - scikit-learn
+    - jupyter
+    - notebook
+    - jupyterlab
+test:
+  imports:
+    - glaes
+  commands:
+    - pip check
+    - python -m pytest 
+  source_files:
+    - .
+  requires:
+    - pip
+    - pytest
+    - pytest-cov
+
+about:
+  home: https://github.com/FZJ-IEK3-VSA/glaes
+  license: MIT
+  license_file:
+    - LICENSE.txt
+  summary: GLAES is a framework for conducting land eligibility analyses and is designed to easily incorporate disparate geospatial information from a variety of sources into a unified solution.
```

Comment:
This summary is quite long! Why not use the upstream one: `Geospatial Land Availability for Energy Systems`.

You can add a multi-line `description` if you want to keep that BTW.

Reply:
Good suggestion. I was not sure about the appropiate length